.github/workflows/python-app.yml

```diff
@@ -52,6 +52,10 @@
        run: poetry install --no-interaction --no-root
      - name: Install
        run: poetry install --no-interaction
+     - name: Manually install previously yanked versions of torch and torchvision
+       run: | 
+         poetry run python -m pip install torch==1.12.0
+         poetry run python -m pip install torchvision==0.2.2.post3
```

Comment:
This is the hacky workaround (for test purposes only) -- the versions of things are still there.  Remove them from the poetry file, and manually install them with pip instead.  Though they've been yanked, pip still finds them.  They were yanked because they don't work with python3.11, which doesn't affect us currently.

Reply:
Fair enough. Not pretty but it works.